### DISCUSSION TITLE
Lock computer on remote connection

### DISCUSSION THREAD

**Discussion by Raito00** [Ylianst/MeshCentral#5616] [Q&A]
Hi!
There is config option when connect to device desktop it first logout from user or lock screen?

**Comment by si458** [comment]
No the isn't, when u connect u see whatever is on the remote device
But
When you right click the disconnect button u can disconnect+logout

**Comment by Raito00** [comment]
This additional option could be added in the future ? 

Because it is necessary in compliance with GDPR regulations.

**Reply by silversword411** [reply]
Can you paste the relevant GDPR language you're basing that off of?
